### PR TITLE
systemd: logs: fix priority when 'Only Emergency' was selected from the UI

### PR DESCRIPTION
**Pull request by KKoukiou**
'Only Emergency' priority maps to priority level 0. Fix the condition to
check if priority level is a number instead of checking if prio_level is
truthy.

https://bugzilla.redhat.com/show_bug.cgi?id=1665421

### CODE REVIEW

File: pkg/systemd/logs.js [hunk 1]
```diff
@@ -288,9 +288,11 @@ $(function() {
             }
         }
 
-        if (prio_level) {
+        if (!isNaN(prio_level)) {
             for (var i = 0; i <= prio_level; i++)
                 match.push('PRIORITY=' + i.toString());
+        } else {
+            console.warn("Priority level was incorrectly set");
```
Comment (martinpitt) [inline pkg/systemd/logs.js]: Maybe actually print the value here, so that it's easier to debug?

Comment (KKoukiou) [reply pkg/systemd/logs.js]: Done

**Comment by martinpitt** [comment]
Thanks, this looks great now, including the new test! That also works everywhere, but it seems this change breaks the
[testAbrtSegv](https://fedorapeople.org/groups/cockpit/logs/pull-10951-20190111-175454-0394147c-verify-rhel-8-0/log.html#212) test on most OSes -- possibly because of the default filter, and this doesn't include crashes any more?

File: pkg/systemd/logs.js [hunk 2]
```diff
@@ -288,9 +288,11 @@ $(function() {
             }
         }
 
-        if (prio_level) {
+        if (!isNaN(prio_level)) {
             for (var i = 0; i <= prio_level; i++)
                 match.push('PRIORITY=' + i.toString());
+        } else {
+            console.warn('Priority level was incorrectly set to ', prio_level);
```
Comment (KKoukiou) [inline pkg/systemd/logs.js]: I 'll remove this warning. In case priority was Everything prio_level is not a number but still valid.

**Comment by KKoukiou** [comment]
testAbrtSegv is crashing because it's expecting
'crashed in os_kill' but the logs contain 'crashed in ??'
Apparently python3-debuginfo is missing, but I don't understand how my changes potentially could affect this.

**Comment by KKoukiou** [comment]
So, I had by mistake seperated the skipImage decorators from the testAbrtSegv test and we didn't see it nicely from the diff. Reposted and moved them where they should be :)

**Pull request merged**